test/src/simple_open_auction.vy

```diff
@@ -0,0 +1,85 @@
+# Open Auction
```

Comment:
let's factor this auction out into a separate repo and make it private until we are sure we can endorse it, I feel it is too complex to put into a release like it's no big deal

Reply:
nvm see other comment, let's just make a much much simpler example to ship with the core thing